modules/developers-guide/pages/cli-reference/dfx-envars.adoc

```diff
@@ -7,6 +7,22 @@ You can configure certain properties for your {sdk-short-name} execution environ
 
 This section lists the environment variables that are currently supported with examples of how to use them.
 
+== CANISTER_CANDID_PATH_{canister.name}
+
+Use the `CANISTER_CANDID_PATH` environment variable to reference the path to the Candid description file for the canisters that are listed as dependencies in the `dfx.json` file for your project.
+
+For example, if you have a `whoami_assets` canister that lists `whoami` under the `dependencies` key, you could use the `CANISTER_CANDID_PATH_whoami_assets` environment variable to refer to the location of the `whoami.did` file, which for local development might be:
+
+....
+$PROJECT_ROOT/.dfx/local/canisters/whoami/whoami.did
+....
+
+== CANISTER_ID_{canister.name}
+
+Use the `CANISTER_ID` environment variable to reference the canister identifier for each canister in the `dfx.json` file for your project.
```

Comment:
In context it makes sense, but based just on scanning, I'd read this as assuming there's an environment variable called `CANISTER_ID`, when usually there won't be one exactly named like that. There will be some *prefixed with* `CANISTER_ID_`....